frontend/theme/theme.ts

```diff
@@ -0,0 +1,7 @@
+import { extendTheme } from "@chakra-ui/react";
+
+export default extendTheme({
+  colors: {
+    lightBlue: "#2d9cdb",
+  },
+}) as Record<string, unknown>;
```

Comment:
Love this! We may have merge conflicts depending on whether this pr or #30 gets merged first, which shouldn't be too much of a fix

Suggestion: After snooping through the chakra ui types, it seems that the theme obj is type-defined as Dict from `@chakra-ui/utils`, do you think it would be a good idea to change the type of this obj? I was wondering if it made a difference